src/main/kotlin/no/nav/syfo/brev/narmesteleder/domain/NarmesteLederBrevDTO.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.syfo.brev.narmesteleder.domain
+
+import no.nav.syfo.dialogmote.domain.DocumentComponentDTO
+import java.time.LocalDateTime
+
+data class NarmesteLederBrevDTO(
+    val uuid: String,
+    val deltakerUuid: String,
```

Comment:
Kanskje vært tydeligere dersom denne hadde vært kalt arbeidstakerUuid?

Reply:
Jeg er usikker på om det er et behov for det, men kunne vurdere å legge til deltakerArbeidsgiverUuid i tillegg.